db/migrate/20150608190453_create_results.rb

```diff
@@ -0,0 +1,17 @@
+class CreateResults < ActiveRecord::Migration
+  def change
+    create_table :results do |t|
+      t.integer :assessment_id
+      t.string :assessment_type
+      t.string :name
+      t.string :description
+      t.string :problem_description
```

Comment:
Should all three of these fields (name, description, problem description) editable in the result? What are some actual examples of what these values would be? Any chance we can find better names than `description` and `problem_description`? I'm just not sure how they are supposed to differ.